packages/oracle/contracts/interfaces/IOracle.sol

```
@@ -53,4 +53,5 @@ interface IOracle is IOracleProvider, IInstance {
     function withdraw(Token18 token) external;
     function beneficiary() external view returns (address);
     function global() external view returns (OracleGlobal memory);
+    function oracles(uint256) external view returns (IOracleProvider provider, uint96 timestamp);
```

It's better to surface the full struct in these kinds of getters instead of the default tuple that solidity does.

So for this one we'd want to:
- change this return type to `Epoch`
- change the storage variable to `private _oracles`
- create a getter function called `external oracles()`